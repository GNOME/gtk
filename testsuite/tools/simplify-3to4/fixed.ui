<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkFixed">
    <property name="visible">True</property>
    <child>
      <object class="GtkButton" id="button">
      </object>
      <packing>
        <property name="x">10</property>
        <property name="y">15</property>
      </packing>
    </child>
  </object>
</interface>
