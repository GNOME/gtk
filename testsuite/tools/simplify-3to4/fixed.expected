<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkFixed">
    <child>
      <object class="GtkButton" id="button">
        <layout>
          <property name="transform">translate(10, 15)</property>
        </layout>
        <property name="visible">0</property>
      </object>
    </child>
  </object>
</interface>
